mbed_lstools/lstools_base.py

```diff
@@ -308,23 +324,27 @@ def get_string(self, border=False, header=True, padding_width=1, sortby='platfor
 
     # Private functions supporting API
 
+    @deprecated("Using internal functions is deprecated.")
```

Comment:
For this particular deprecated message I think I'd give a bit more of a warning/reason.

"This function will be removed from the public API in future releases. Please use 'list_mbeds' instead."